presto-main/src/main/java/io/prestosql/failuredetector/HeartbeatFailureDetector.java

```diff
@@ -221,6 +222,75 @@ public int getActiveCount()
         return builder.build();
     }
 
+    @Override
+    public void updateMonitorServices()
```

Comment:
refactor updateMonitoredServices() so that the common functionality is not duplicated